xtable-utilities/src/main/resources/xtable-hadoop-defaults.xml

```diff
@@ -55,15 +55,15 @@
   </property>
   <property>
     <name>fs.s3.aws.credentials.provider</name>
-    <value>com.amazonaws.auth.DefaultAWSCredentialsProviderChain</value>
+    <value>software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider</value>
```

Comment:
It could be useful to pull the aws and hadoop upgrade related code into a smaller PR that can be merged quickly for anyone that may be running into the compatibility issues you saw with hadoop and aws sdk

Reply:
PR for aws sdk and hadoop version upgrade: https://github.com/apache/incubator-xtable/pull/614